### ISSUE TITLE
Calendar - should be able to set the `level` of the header

### ISSUE THREAD

**Issue by jcfilben** [grommet/grommet#7450]
### Expected Behavior
A user should be able to pass `level` to the Calendar component to control the heading level used.

### Actual Behavior
Heading level can't be controlled

### Steps to Reproduce

1. 2. 3.

### Your Environment

- Grommet version:
- Browser Name and version:
- Operating System and version (desktop or mobile):
